### ISSUE TITLE
Anomaly value - loss

### ISSUE THREAD

**Issue by XUYIRS** [avanetten/yoltv4#6]
When I train the model with rareplane dataset, it reported that "v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 139 Avg (IOU: 0.307299), count: 23, class_loss = 8190.737305, iou_loss = 33.274414, total_loss = 8224.011719 "
Why does it has so large loss?
And after several times, it changed to "v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 150 Avg (IOU: 0.000000), count: 18, class_loss = -nan, iou_loss = -nan, total_loss = -nan", all the loss values changed to -nan.
I have tried to change many parameters, but it did not work either.
![Screenshot from 2021-07-20 17-39-13](https://user-images.githubusercontent.com/31101718/126300308-728c3d1d-d16a-4c40-ba98-9a979a20a879.png)

**Comment by jacksonsc007** [comment]
@XUYIRS,
Hi, I encountered some issues when building with docker and I hope you can help me with that. Basically, it is as same as [this issue](https://github.com/avanetten/yoltv4/issues/15), but I didn't get the answer. Could you help me with that?
